core/src/main/java/cz/fi/muni/pv217/employee/management/entity/Workday.java

```diff
@@ -0,0 +1,24 @@
+package cz.fi.muni.pv217.employee.management.entity;
+
+import io.quarkus.hibernate.orm.panache.PanacheEntity;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+
+@Entity
+public class Workday extends PanacheEntity {
+    @NotNull(message = "Date cannot be null")
+    @Column(columnDefinition = "DATE")
+    public LocalDate date;
+
+    public Double hours;
```

Comment:
Hours can be simply int too, you usually ceil the hours.

Reply:
Because employee can work 7 hours and 30 minutes, which is 7.5 h.